router.go

```diff
@@ -326,20 +330,35 @@ func (r *Router) GetVersion() string {
 	return r.version
 }
 
+// DefaultBodyReadTimeout sets the amount of time an operation has to read
+// the body of the incoming request before it is aborted. Defaults to 15
+// seconds if not set.
+func (r *Router) DefaultBodyReadTimeout(timeout time.Duration) {
```

Comment:
I would recommend setting defaults here if the caller doesn't pass in a value.

Reply:
This (and the related func below) is an optional setter so it may not ever get called, and passing in a zero value is a valid way to disable the default timeout.

Reply:
So is the only way to set a default value now in the `&Router{}` configuration? If defaults aren't included there, is the expected behavior to wait indefinitely?

Reply:
Spoke in person and we agreed this will work for now to help do some debugging.